img/foxsi_image_map.pro

```diff
@@ -56,12 +56,15 @@ default, fov, 20.
 	time = anytim( time, /yo )
 
 	; Basic image production
-	if keyword_set( CDTE ) then $
-		image = foxsi_image_det_cdte( data, trange=trange, erange=erange, $
-						keepall=keepall, year=year, thr_n=thr_n )	$
-		else $
+;	if keyword_set( CDTE ) then $
```

Comment:
Any reason to keep these lines commented in the code?

Reply:
No, I removed it in my version, I hope it will show in the pull request.